R/zzz.R

```diff
@@ -1,5 +1,27 @@
 .lockbox_env <- new.env()
 
+.get_safely_from_envir <- function(target, default = NULL) {
+  res <- tryCatch(
+    get(target, envir = .lockbox_env)
+  , error = function(e) {default})
```

Comment:
Agree that the multiple equivalents are confusing. I typically only find myself using `get` if I need to specify `inherits = FALSE`.